src/consensus/aft/test/main.cpp

```diff
@@ -1281,6 +1278,7 @@ DOCTEST_TEST_CASE(
           DOCTEST_REQUIRE(msg.term == 2);
           DOCTEST_REQUIRE(msg.last_commit_idx == 1);
           DOCTEST_REQUIRE(msg.last_commit_term == 1);
+          DOCTEST_REQUIRE(msg.last_committable_idx == 2);
```

Comment:
This is only true because `committable` is set to true in `DOCTEST_REQUIRE(r0.replicate(kv::BatchVector{{2, second_entry, true}}, 1));`. I wonder if we should replicate another non-committable entry after 2?

Reply:
At the moment the raft tests only deal with committable indices (because the notion of uncommittable index is alien to raft). I don't want to extend them too much because they are quite impractical, and I would like to replace them with a better interface instead.